test/GPv2Settlement/InvalidateOrder.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8.26;
+
+import {Vm} from "forge-std/Test.sol";
+
+import {GPv2Order} from "src/contracts/libraries/GPv2Order.sol";
+
+import {Helper, GPv2Settlement} from "./Helper.sol";
+
+//solhint-disable func-name-mixedcase
```

Comment:
This is what I meant when I mentioned that we want different Solhint configs for `tests` and `src`. In principle we should add this to every test file, and this wouldn't be the only config line. Note that this can't be done with just config files (see [here](https://github.com/protofire/solhint/issues/), issue 577).
Not related to this PR, just something to remember once we rework how Solhint is configured.